instagram/src/components/PostContainer/Likes.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import styled from "styled-components";
+
+const LikesWrapper = styled.div`
+
+`;
+
+const SocialWrapper = styled.div`
+    display: flex;
+`;
+
+const Social = styled.div`
+    display: flex;
+`;
+
+const Lykes = styled.h2`
+  font-weight: bold;
+`;
+
+const Likes = props => {
+  return (
+    <LikesWrapper>
+        <SocialWrapper onClick = {props.incrementHandler}>
+      <Social>
```

Comment:
```suggestion
        <Social>
```
Watch out on your indentations